app/assets/javascripts/angular/components/surveyOverviewQuestionsTab/surveyOverviewQuestions.component.js

```diff
@@ -260,7 +260,9 @@ function surveyOverviewQuestionsController(
             component: 'predefinedQuestionsModal',
             size: 'md',
             resolve: {
-                currentQuestions: () => this.surveyQuestions.map(q => q.id),
+                orgId: () => this.survey.organization_id,
+                currentQuestions: () =>
+                    this.surveyQuestions.map(question => question.id),
```

Comment:
Looks like I changed this back in a merge conflict resolution... Looks like you shortened the name to fit it on one line. I don't care that much. I like meaningful names but this is trivial. I could undo that.

I realized a good practice might be that if we don't care enough about a variable to give it a meaningful name, maybe we should destructure it directly. I guess that could depend on context but it seems like it would work with simple things.
```suggestion
                    this.surveyQuestions.map(({ id }) => id),
```

Reply:
yup i agree, the substructure is  a nice touch :)